src/towncrier/_writer.py

```diff
@@ -35,11 +35,12 @@ def append_to_newsfile(
 
         if len(existing_content) > 1:
             f.write(existing_content.pop(0).rstrip().encode("utf8"))
-            if start_line:
-                f.write((u"\n\n" + start_line + u"\n").encode("utf8"))
+            if start_string:
+                f.write((u"\n\n" + start_string + u"\n").encode("utf8"))
 
         f.write(top_line.encode("utf8"))
         f.write(content.encode("utf8"))
-        if existing_content[0]:
-            f.write(b"\n\n")
-        f.write(existing_content[0].lstrip().encode("utf8"))
+        if existing_content:
+            if existing_content[0]:
+                f.write(b"\n\n")
+            f.write(existing_content[0].lstrip().encode("utf8"))
```

Comment:
This is probably better fixed by setting `existing_content = [u"", u""]` above when `single_file` is `False`.

```suggestion
        if existing_content[0]:
            f.write(b"\n\n")
        f.write(existing_content[0].lstrip().encode("utf8"))
```